stl/inc/future

```diff
@@ -141,6 +138,11 @@ private:
     error_code _Mycode; // the stored error code
 };
 
+// TRANSITION, ABI: remove `_v2` suffix
+[[noreturn]] inline void _Throw_future_error_v2(const future_errc _Errno) {

```

Comment:
Is there a reason this `future_errc` is called `_Errno`? That seems really odd to me given that it's, afaik, not taken from `errno`...

edit: it looks like other places use the `_Errno` name, nvm. Still weird.

Reply:
I suspect the reason is "because a `future_errc` is a sort of error number like `errno` is." Feel free to file a cleanup issue to change all of these consistently to something better.

Reply:
Yeah, this is weird, especially because elsewhere we have `_Errno_ref` bound to the actual `errno`. I can just clean it up now (9 occurrences is small). I've used `_Ec` for `errc` variables in the past:
https://github.com/microsoft/STL/blob/314f65f831f827dbfb05ebcc0f08c496f5ddf64f/stl/inc/charconv#L1861-L1865